src/features/instance/ProcessContext.tsx

```diff
@@ -116,6 +133,37 @@ export function useRealTaskType() {
   return useRealTaskTypeById(taskId || undefined);
 }
 
+export function useTaskType(taskId: string | undefined) {
```

Comment:
Maybe we should have some comments here to describe the difference between all these similarly named hooks.